sqlglot/dialects/sqlite.py

```diff
@@ -186,6 +189,7 @@ class Generator(generator.Generator):
                     transforms.eliminate_semi_and_anti_joins,
                 ]
             ),
+            exp.StrPosition: strposition_to_instr_sql,
```

Comment:
We'll remove this after the suggested refactor is implemented.

Reply:
I replaced this with a lambda call to `dialect->str_position_sql`